README.rst

```diff
@@ -104,7 +109,7 @@ Checklist module
 Meta
 ====
 
-* License: MIT, see `LICENSE file <LICENSE>`__
+* License: MIT,  `see here <LICENSE>`__
```

Comment:
I prefer to be explicit, that is the name of the file and the link to it. 

```suggestion
* License: MIT,  `see LICENSE file <LICENSE>`__
```

Reply:
Noted!